hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -315,10 +315,41 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
             assets_df = pd.DataFrame(data=assets_data, columns=assets_columns)
             lines.extend(["", "  Assets:"] + ["    " + line for line in str(assets_df).split("\n")])
 
-            bid_profitability, ask_profitability = self.c_calculate_market_making_profitability(market_pair,
-                                                                                                maker_order_book,
-                                                                                                taker_order_book)
-            lines.extend(["", "  Profitability:"] +
+            bid_price, bid_size_limit = self.c_get_market_making_price_and_size_limit(
+                market_pair,
+                True,
+                own_order_depth=0
+            )
+            bid_size = self.c_get_adjusted_limit_order_size(
+                        market_pair,
+                        float(bid_price),
+                        float(bid_size_limit)
+                    )
+
+            ask_price, ask_size_limit = self.c_get_market_making_price_and_size_limit(
+                market_pair,
+                False,
+                own_order_depth=0
+            )
+            ask_size = self.c_get_adjusted_limit_order_size(
+                        market_pair,
+                        float(ask_price),
+                        float(ask_size_limit)
+                    )
+            bid_profitable = self.c_has_market_making_profit_potential(
```

Comment:
c_has_market_making_profit_potential returns a boolean tuple, but the standard across strategies is to show profitability as a number. 

Please incorporate fees to c_calculate_market_making_profitability instead